### ISSUE TITLE
mismatched_organization error

### ISSUE THREAD

**Issue by yaqinamy** [Azure-Samples/ai-rag-chat-evaluator#76]
ERROR    Failed to send a test chat completion to the GPT deployment due to error:                                                                    evaluate.py:110
                    Error code: 401 - {'error': {'message': 'OpenAI-Organization header should match organization for API key', 'type': 'invalid_request_error',                
                    'param': None, 'code': 'mismatched_organization'}

**Comment by yaqinamy** [comment]
Yes Pamela, I am learning from your video using llama2 via ollama which I deployed local and GPT4 deployed at azure.

**Comment by pamelafox** [comment]
Okay, so to clarify:

* You are using GPT4 for the evaluation?
* You are using llama2 for the chat application itself?


The error in your report shows an issue with the evaluation model, which would be GPT-4. But you shouldn't need to specify OpenAI organization for an Azure model. Do the logs indicate whether it's trying to use Azure or OpenAI.com? 

You should see the logs from this code:

https://github.com/Azure-Samples/ai-rag-chat-evaluator/blob/7b5d503dc96a5185446b16bd8db24558614d2ec9/scripts/service_setup.py#L14
